clang/include/clang/CIR/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+set(MLIR_MAIN_SRC_DIR ${LLVM_MAIN_SRC_DIR}/../mlir/include ) # --src-root
```

Comment:
This variable seems unused and has the same value as `MLIR_INCLUDE_DIR` below, what's the use case for this variable?